modules/openapi-generator/src/test/java/org/openapitools/codegen/OpenAPINormalizerTest.java

```diff
@@ -622,6 +627,11 @@ public void testOpenAPINormalizerProcessingArraySchema31Spec() {
         assertEquals(((Schema) schema6.getProperties().get("arrayOfStrings")).getItems().getTypes().contains("string"), true);
         assertEquals(((Schema) schema6.getProperties().get("arrayOfStrings")).getItems().getType(), "string");
         assertEquals(((Schema) schema6.getProperties().get("arrayOfStrings")).getType(), "array");
+
+        Schema schema8 = openAPI.getComponents().getSchemas().get("ArrayWithPrefixItems");
+        assertNotEquals(((Schema) schema8.getProperties().get("with_prefixitems")).getItems(), null);
```

Comment:
OOI, why not:
```suggestion
        assertNotNull(((Schema) schema8.getProperties().get("with_prefixitems")).getItems());
```
?

Reply:
to me, it's much easier to read when comparing 2 values

Reply:
👍 Interesting, for me it's the opposite, scanning the assertion function first. Also, some of the convenience functions have more precise error messages in case the assertion fails, which I always appreciate whilst sorting out any messes I may have made :)